Doc/library/dataclasses.rst

```diff
@@ -96,8 +96,15 @@ Module contents
      generated.  The generated repr string will have the class name and
      the name and repr of each field, in the order they are defined in
      the class.  Fields that are marked as being excluded from the repr
-     are not included.  For example:
-     ``InventoryItem(name='widget', unit_price=3.0, quantity_on_hand=10)``.
+     are not included.  For example::
+
+       @dataclass
+       class InventoryItem:
+           name: str = 'widget'
+           unit_price: float = 3.0
+           quantity_on_hand: int = field(default=10, repr=False)
+
+     ``InventoryItem(name='widget', unit_price=3.0)``.
```

Comment:
You should use a doctest to show what the output also looks like. Otherwise it does not really help. In addition, I would have specified this example in the ``repr`` for `fields` and not for the decorator. If you think that the documentation must be improved, you should have two different examples: one with `@dataclass(repr=False)` and one with `field(repr=False)`, e.g.:

```py
>>> @dataclass(repr=False)
... class Vec2D:
... 	x: float
... 	y: float
...
>>> Vec2D(x=0, y=0)  # doctest: +ELLIPSIS
<__main__.Vec2D object at 0x...>
```

and

```py
>>> @dataclass
... class Vec2D:
... 	x: float
... 	y: float
... 	name: str = field(repr=False)
...
>>> Vec2D(x=0, y=0, name='origin')
Vec2D(x=0, y=0)
```